src/tool/cppwinrt/strings/base_identity.h

```diff
@@ -224,8 +224,12 @@ namespace winrt::impl
     template <typename T>
     struct name
     {
+#ifdef __clang__
+        inline static const auto value
```

Comment:
This is as a result of guid_of not always being a constexpr expression when the type is a classic COM interface. See next comment.